WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -581,4 +789,58 @@ public void onPostUploaded(OnPostUploaded event) {
             }
         }
     }
+
+    @Override public void onPositiveClicked(@NonNull String instanceTag) {
+        switch (instanceTag) {
+            case TAG_ADD_SITE_ICON_DIALOG:
```

Comment:
Both `TAG_ADD_SITE_ICON_DIALOG` and `TAG_CHANGE_SITE_ICON_DIALOG` do the same thing so we can consolidate them into this.
```
case TAG_ADD_SITE_ICON_DIALOG:
case TAG_CHANGE_SITE_ICON_DIALOG:
    ActivityLauncher.showPhotoPickerForResult(getActivity(),
            MediaBrowserType.SITE_ICON_PICKER, getSelectedSite());
    break;
```